packages/airnode-node/package.json

```diff
@@ -29,8 +29,8 @@
     "@api3/airnode-protocol": "^0.13.0",
     "@api3/airnode-utilities": "^0.13.0",
     "@api3/airnode-validator": "^0.13.0",
-    "@api3/commons": "^0.3.0",
-    "@api3/ois": "2.2.1",
+    "@api3/commons": "file:../../../commons",
+    "@api3/ois": "file:../../../ois",
```

Comment:
Want me to release OIS v2.3 now that https://github.com/api3dao/ois/pull/117 is merged?

Reply:
I've asked for npm access so I might do it myself.